pkg/lib/web/static_assets.go

```diff
@@ -31,6 +32,38 @@ var AuthgearCSS = resource.RegisterResource(CSSDescriptor{
 	Path: StaticAssetResourcePrefix + "authgear.css",
 })
 
+var NormalizeCSS = resource.RegisterResource(CSSDescriptor{
+	Path: StaticAssetResourcePrefix + "normalize.min.css",
+})
+
+var NormalizeCSSMap = resource.RegisterResource(resource.SimpleDescriptor{
+	Path: StaticAssetResourcePrefix + "normalize.min.css.map",
+})
+
+var IconsCSS = resource.RegisterResource(CSSDescriptor{
```

Comment:
I found I cannot simply change `CSSDescriptor` to `resource.SimpleDescriptor`, as their `ViewResources` functions are returning different object. `CSSDescriptor` returns struct `web.StaticAsset` which has the path and data. But `resource.SimpleDescriptor` returns data directly. 

When I called `$.StaticAssetURL` in the template, the function expect the descriptor return `web.StaticAsset` but `resource.SimpleDescriptor` doesn't.

Should we create another `SimpleDescriptor` under `/pkg/lib/web/` that will return `web.StaticAsset`, but then we will need to copy the logic from `pkg/util/resource`. Or just keep using the `CSSDescriptor` to avoid too many descriptors implementation?

Reply:
Just keep using CSSDescriptor is fine.

Reply:
Then the PR is good to go:)